etna/articles/models.py

```diff
@@ -369,7 +373,7 @@ class RecordArticlePage(
     TopicalPageMixin, ContentWarningMixin, NewLabelMixin, BasePageWithIntro
 ):
     template = "articles/record_article_page.html"
-    parent_page_types = ["collections.ExplorerIndexPage"]
+    parent_page_types = ["collections.ExplorerIndexPage", "articles.ArticleIndexPage"]
```

Comment:
I would just go with the following here:
```suggestion
    parent_page_types = ["articles.ArticleIndexPage"]
```

These attributes are only looked at when adding new or moving existing pages - If we don't want articles to live under the explorer index page any longer, we shouldn't include it.

Reply:
Hi Andy, sorry - I just messaged you on Slack about this one 😄 


I've left the original parent/sub-page types in because when I tried doing "move" on my local after removing the original parent/sub-page type I kept getting an error saying the page didn't exist or something to that effect

Reply:
NOTE: Resolved on slack - this is temporary due to the error/problem mentioned above, an update will be put in for the next deployment (Tues 27th) to remove this